src/bin/helpers.js

```diff
@@ -433,6 +433,21 @@ function toUpperCamelCase (str) {
   }).replace(/\s+/g, "")
 }
 
+function* toUtf16Bytes(str) {
+  for (let i = 0; i < str.length; i++) {
+    const charCode = str.charCodeAt(i);
+
+    // Get the most significant byte.
+    // For example, given 0x1234, yield 0x12.
+    yield (charCode & 0xff00) >> 8;
+
+    // Get the least significant byte.
+    // For example, given 0x1234, yield 0x34.
+    yield charCode & 0x00ff;
+  }
+}
+
+
```

Comment:
As a rule of thumb, I'd avoid exporting a method that is a generator because it could be difficult to understand. Is there any advantage here? I think this is an equivalent version
```suggestion
function toUtf16Bytes(str) {
  const bytes = new Uint8Array(str.length * 2);
  for (let i = 0; i < str.length; i++) {
    const code = str.charCodeAt(i);
    bytes[i * 2] = code >> 8;
    bytes[i * 2 + 1] = code & 0xff;
  }
  return bytes;
}
```